invenio_communities/communities/resolver.py

```diff
@@ -27,15 +27,38 @@
 from .services.config import CommunityServiceConfig
 
 
+def tombstone_community(id):
+    """Return a tombstone community."""
+    return {
```

Comment:
moderate: this is hardcoding a dict/datamodel, would it be worth it to use [CommunitySchema](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/communities/schema.py#L114), or at least dump it to make sure we comply?